websites/A/AnimeGO/presence.ts

```diff
@@ -0,0 +1,142 @@
+const presence = new Presence({
+	clientId: "935597176426491924"
+});
+
+async function getStrings() {
+	return presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused"
```

Comment:
```suggestion
		play: "general.playing",
		pause: "general.paused"
```

Reply:
Do these strings really need to be changed? In this case, the strings are not suitable. If you use these strings, then in russian it will be written that the person is playing the game, and not watching the video.

Reply:
Yes, these strings may be removed in the future.

Reply:
> Do these strings really need to be changed? In this case, the strings are not suitable. If you use these strings, then in russian it will be written that the person is playing the game, and not watching the video.

Can't you just make your own strings without `getString` function?